sample-compose/data/src/main/java/co/nimblehq/sample/compose/data/response/ErrorResponse.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.sample.compose.data.response
+
+import co.nimblehq.sample.compose.domain.model.Error
+import com.squareup.moshi.Json
+
+data class ErrorResponse(
+    @Json(name = "message")
+    val message: String
+)
+
+internal fun ErrorResponse.toModel() = Error(message = message)
```

Comment:
Then we can get rid of this `toModel` extension.

Reply:
@lydiasama explained above ☝️